mkdocs_puml/plugin.py

```diff
@@ -67,10 +71,11 @@ def on_config(self, config: Config) -> Config:
         self.puml = PlantUML(
             self.config['puml_url'],
             num_workers=self.config['num_workers'],
-            verify_ssl=self.config['verify_ssl']
+            verify_ssl=self.config['verify_ssl'],
         )
         self.puml_keyword = self.config['puml_keyword']
-        self.regex = re.compile(rf"```{self.puml_keyword}(.+?)```", flags=re.DOTALL)
+        self.regex = re.compile(rf"```{self.puml_keyword}(\n.+?)```", flags=re.DOTALL)
```

Comment:
I want to point out this change, I updated the regex here to fix a bug.

Assume our `puml_keyword` is `puml` the previous implementation would match it in ex `puml_light` and `puml_dark` (I was testing something). 

This results in getting bad content of the puml since it would have an additional `_dark` or `_light`  before the `@startuml`

The new regex would match until end of line so it doesn't have this issue :+1: